src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -848,8 +848,14 @@ public final HttpClient wiretap(boolean enable) {
 
 	/**
 	 * Specifies whether the metrics are enabled on the {@link HttpClient}.
-	 * All generated metrics are registered in the Micrometer MeterRegistry,
+	 * All generated metrics are registered in the global composite registry
+	 * provided by Micrometer with name prefix {@code "reactor.netty.http.client"},
 	 * assuming Micrometer is on the classpath.
+	 * <p>
+	 * Note: As the the global composite registry is used for registering the meters,
+	 * all MeterFilter (for example for limiting the maximum allowable tags should be applied
+	 * outside Reactor Netty.
+	 * </p>
```

Comment:
The note seems incomplete (and a closing parenthesis is missing). Perhaps:

```java
/**
 * Whether to enable metrics to be collected and registered in Micrometer's
 * {@link io.micrometer.core.instrument.Metrics#globalRegistry globalRegistry}
 * under the name {@code "reactor.netty.http.client"}. Applications can
 * separately register their own
 * {@link io.micrometer.core.instrument.Meter meters} associated with this name.
 * For example, to put an upper bound on the number of tags produced:
 * <pre class="code">
 * String name = "reactor.netty.http.client"
 * MeterFilter filter = ... ;
 * Metrics.globalRegistry.config().meterFilter(
 * 		MeterFilter.maximumAllowableTags(name, 100, filter));
 * </pre>
 * <p>By default this is not enabled.
 * @param metricsEnabled true enables metrics collection; false disables it
 * @return a new {@link HttpClient}
 */
```

Reply:
done